modules/steam.nix

```diff
@@ -45,6 +45,17 @@ in
             Use `systemctl --user stop steam.target` to stop.
           '';
         };
+        enableUdevRules = mkOption {
+          type = types.bool;
+          default = true;
+          description = ''
+            Whether to make certain device attributes controllable by users.
+
+            The Steam Deck Client directly modifies several device attributes to
+            control the display brightness and to enable performance tuning (TDP
+            limit, GPU clock control).
+          '';
+        };
```

Comment:
(Nothing really to do here)

I think this is fine right now as a description. In the future it could include other steam-shipped udev rules. There is nothing intrinsically opinionated for the steam deck. All of the rules are rules that open control for the end-user. At worst, the rules are present on hardware where they don't get triggered.